### DISCUSSION TITLE
ravu-zoom-r3和KrigBilateral同时使用会多执行一次chroma upscale

### DISCUSSION THREAD

**Discussion by yeezylife** [hooke007/mpv_PlayKit#200] [mpv 通用]
![侧视图](https://user-images.githubusercontent.com/30175425/197309950-752ebbf2-74dc-462f-903e-3cb5ea8a8721.jpg)

小比例的放大想交给ravu-zoom执行，于是从superres换回ruvu-zoom看了看，结果发现ravu-zoom-r3和KrigBilateral同时使用会多执行一次chroma upscale的情况。如图所示。

怎么会这样呢 这是普遍问题么 我记得以前用vo=gpu的时候没这种现象呀，难道是gpu-next的问题？

**Comment by yeezylife** [comment]
我是不是应该用ravu-zoom-rgb？ 这样就能在KrigBilateral之后再生效了。。。ravu-zoom-rgb和ravu-zoom-yuv应该选哪个？
介绍里写着“-yuv  无法处理rgb的源”  那  -rgb是不是也无法处理 yuv的源呢 是不是两个变体都要写进配置里。。

**Reply by hooke007** [reply]
> 多执行一次chroma upscale

...这种问题不应该问mpv issue吗

**Reply by yeezylife** [reply]
我怕是自己下错变体了。算了用回superres就好，ravu能效比挺好的，可惜不怎么更新了。

**Reply by hooke007** [reply]
> -rgb是不是也无法处理 yuv的源呢

`-yuv`无法处理rgb的源 是因为 hook 的stage 是YUV→RGB的矩阵转换前的步骤
`-rgb` hook 的就是 转换到 RGB 的步骤
